resources/lang/fr.json

```diff
@@ -195,7 +195,7 @@
     "A meeting title must be provided in at least one language.": "Un titre de réunion doit être fourni dans au moins une langue.",
     "American Sign Language (ASL)": "American Sign Language (ASL)",
     "Analyze the outcomes of your consultation with people who are in the context of disability and Deaf groups.": "Analyser les résultats de votre consultation avec des personnes faisant partie de groupes de personnes en situation de handicap et de personnes sourdes.",
-    "An approximate response time must be provided in at least one language.": "",
+    "An approximate response time must be provided in at least one language.": "An approximate response time must be provided in at least one language.",
```

Comment:
This is in English. Shouldn't it be an empty string if no translation is available?

Reply:
This would be the same for the other English strings present.